src/client/components/search-component/index.scss

```diff
@@ -0,0 +1,109 @@
+.hidden {
```

Comment:
One of the benefits of using SASS is the ability to nest your rules. Read more https://sass-lang.com/guide (Nesting)
Also, it's better to use just one class that is very specific (by using BEM) that having a chain of classes. There more selectors you use in CSS the lower your performance will be.